src/main.rs

```diff
@@ -2,21 +2,25 @@ mod collectors;
 mod config;
 mod error;
 mod exporters;
+mod logging;
 mod message;
 mod uption;
 mod url;
 
 use std::process;
 
+use log::{error, info};
+
 use crate::config::UptionConfig;
 use uption::Uption;
 
 fn main() {
+    logging::bootstrap_logging();
     set_ctrl_c_handler();
 
-    println!("Reading configuration");
+    info!("Reading configuration");
     let config = UptionConfig::new().unwrap_or_else(|err| {
```

Comment:
Chicken-egg problem: If the logging bootstraps before config, then we can't insert config to logging. Otherwise we can't log anything else before the config is ready.

Reply:
I think we can just remove that reading configuration line and start logging after configuration is read.

Reply:
One option is to bootstrap logging with some defaults (stdout, info level)  and then read the configuration which could modify logging settings.